creator-node/src/logging.js

```diff
@@ -1,5 +1,8 @@
 const bunyan = require('bunyan')
 const shortid = require('shortid')
+const { createNamespace } = require('cls-hooked')
+
+const namespace = createNamespace('namespace')
```

Comment:
this file is a bit confusing to me, might be helpful to discuss briefly
would be good to have a file comment summarizing
i'm down to write that up after we discuss / i understand haha

Reply:
sounds good, will add a comment if we decide to keep it